pkg/apis/core/validation/cloudprofile.go

```diff
@@ -189,6 +189,12 @@ func validateMachineImages(machineImages []core.MachineImage, fldPath *field.Pat
 			if err != nil {
 				allErrs = append(allErrs, field.Invalid(versionsPath.Child("version"), machineVersion.Version, "could not parse version. Use SemanticVersioning. In case there is no semVer version for this image use the extensibility provider (define mapping in the ControllerRegistration) to map to the actual non-semVer version"))
 			}
+
+			for j, cri := range machineVersion.ContainerRuntimeInterfaces {
+				if !avaliableWorkerCRINames.Has(string(cri)) {
```

Comment:
should we also check CRIs with container runtimes supported / CRI?

Reply:
No, the [GEP](https://github.com/gardener/gardener/blob/master/docs/proposals/10-shoot-additional-container-runtimes.md#proposal) implies that every CR rely on `containerd`.